src/index.html

```diff
@@ -2,12 +2,60 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="./css/style.css">
```

Demo link is empty, try to update path and redeploy your changes
![image](https://github.com/mate-academy/layout_stars/assets/80700247/9006a35a-a26d-4358-a396-f6192a13169a)